recipes/natural_language_processing/chatbot/README.md

```diff
@@ -68,27 +73,72 @@ cd ../recipes/natural_language_processing/chatbot
 _A full list of supported open models is forthcoming._  
 
 
-### Build and Deploy the Model Service
+### Build the Model Service
 
 The complete instructions for building and deploying the Model Service can be found in the
 [llamacpp_python model-service document](../../../model_servers/llamacpp_python/README.md).
 
-The Model Service can be built and ran from make commands from the [llamacpp_python directory](../../../model_servers/llamacpp_python/).
+The Model Service can be built from make commands from the [llamacpp_python directory](../../../model_servers/llamacpp_python/).
 
 ```bash
 # from path model_servers/llamacpp_python from repo containers/ai-lab-recipes
-make -f Makefile build && make -f Makefile run
+make -f Makefile build
+```
+
+or with the podman cmd
+```bash
+podman build -t llamacpp_python . -f  ./base/Containerfile
+``` 
+
+Checkout the [Makefile](../../../model_servers/llamacpp_python/Makefile) to get more details on different options for how to build.
+
+### Deploy the Model Service
+
+The local Model Service relies on a volume mount to the localhost to access the model files. It also employs environment variables to 
+dictate the model used and where its served. You can start your local Model Service using the following `make` or `podman` commands:
+
+```bash
+# from path model_servers/llamacpp_python from repo containers/ai-lab-recipes
+make -f Makefile run
+```
+or 
+```bash
+podman run --rm -it \
```

Comment:
The problem with this is this command can evolve over time and might end up being different then the Makefile command, causing a chore for developers.  But in practice I am not against it.
What do others think?

Reply:
yeah, I see the value of relegating anything changeable to the makefile so it can only break in one place. Just felt like we are masking a lot of functionality/customization not giving the direct podman commands. 

Why don't I revert on this for now, as to not block the other changes in the README. and we can address podman vs makefile questions in a later PR?